app/i18n/cz/conf.php

```diff
@@ -191,6 +191,7 @@
 		'navigation_help' => 'Pomocí přepínače <kbd>⇧ Shift</kbd> fungují navigační zkratky v rámci kanálů.<br/>Pomocí přepínače <kbd>Alt ⎇</kbd> fungují v rámci kategorií.',
 		'navigation_no_mod_help' => 'The following navigation shortcuts do not support modifiers.',	// TODO - Translation
 		'next_article' => 'Skočit na další článek',
+		'non_standard' => 'You are using non-standard shortcut keys. Be aware that you might experience unexpected behaviors. The standard shortcut keys are the following:',	// TODO - Translation
```

Comment:
Since the unexpected behavior is just a harmless not working, I'd strongly reword it so as not to sound frightening:
```suggestion
		'non_standard' => 'Some keys may not work as shortcuts. The standard shortcut keys are the following:',	// TODO - Translation
```